pkg/helm/controller.go

```diff
@@ -34,25 +33,29 @@ var (
 type Controller struct {
 	namespace      string
 	helmController helmcontroller.HelmChartController
+	confController helmcontroller.HelmChartConfigController
 	jobsCache      batchcontroller.JobCache
 	apply          apply.Apply
 }
 
 const (
-	image   = "rancher/klipper-helm:v0.2.7"
-	Label   = "helmcharts.helm.cattle.io/chart"
-	CRDName = "helmcharts.helm.cattle.io"
-	Name    = "helm-controller"
+	image         = "brandond/klipper-helm:v0.3.0"
```

Comment:
can you cut a release of klipper-helm now and update this back to rancher?

Reply:
will do!